File: java-main/java-main/IO/Nestedfor6/p6.java
/*Write a program to take 5 numbers as input from the user and print the count of digits in those number.*/

import java.io.*;
class Demo{
	public static void main(String[] args)throws IOException{
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		System.out.println("How many numbers you want to take from the user?");
		int input=Integer.parseInt(br.readLine());
		int count=0;
		do{
			System.out.println("Enter Number:");
			int num=Integer.parseInt(br.readLine());
			int temp=num;
			int cnt1=0;
			while(temp!=0){
				cnt1++;
				temp=temp/10;
			}
			System.out.println("Number "+num +" has "+cnt1+" digits");
			count++;
		}
		while(count!=input);
	}
}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/practical2/p6.c
#include<stdio.h>
void main(){

	int num;
	int i=1;
	int rem=0,count=0;

	printf("Enter the number\n");
	scanf("%d",&num);

	while(i<=num){
		rem=num%10;
		count++;
		num=num/10;
		i++;
	}
	printf("count is :%d\n",count);
}
